### DISCUSSION TITLE
How to join this project

### DISCUSSION THREAD

**Discussion by 2000liux** [apache/seatunnel#3970] [Q&A] [answered]
- Expect to participate in the development of this project. My can start with documentation and some simple functions
- Join the community mail, or WeChat, DingTalk group, participate in internal discussion meetings, and solve problems together when developing functions
- Please tell me how to do it?

- There has been no news since the subscription was initiated

**Comment by ghost** [comment]
Hello sir ,
I am new to open source contribution.
I already know java , my tech stacks & tools includes C, C++ , Python , Java, JavaScript , HTML , CSS , SQL , Bootstrap, ReactJS, ExpressJS, NodeJS & Git . I need a little help from your side to contribute to these amazing projects.

**Reply by davidzollo** [reply, marked as answer]
Hello, welcome to join the opensource community.
for SeaTunnel Project, you can read the contribution doc firstly, https://seatunnel.apache.org/community/contribution_guide/contribute, then try to do some small things like fixing document spell error or fixing small bugs, don't try to contribute a big PR to the project if you're new, you can do this step by step, first try to do small things, small -> big -> bigger ... . 
I believe you will be great one day. Just try from now, if you meet any problem, you can join our Slack: https://join.slack.com/t/apacheseatunnel/shared_invite/zt-123jmewxe-RjB_DW3M3gV~xL91pZ0oVQ

**Comment by davidzollo** [comment]
@2000liux , welcome to join us!
We have a local community, you can add my wechat: davidzollo, please tell me you'd like to do contributions for Apache SeaTunnel project, I will help you.

**Comment by ghost** [comment]
Yes ,  I am coder , question is simple how to understand these big codebases

**Reply by davidzollo** [reply]
You can start from how to deployment, https://seatunnel.apache.org/docs/2.3.1/start-v2/locally/deployment
then you can try to run the program in your local environment from doc, https://seatunnel.apache.org/docs/2.3.1/contribution/setup
